scene/animation/animation_player.cpp

```diff
@@ -572,7 +572,7 @@ void AnimationPlayer::_animation_process_data(PlaybackData &cd,float p_delta,flo
 
 	} else {
 		
-		if (next_pos<0 or next_pos>len) {
+        if (next_pos<0 || next_pos>len) {
```

Comment:
Most likely your IDE is configured for spaces instead of tabs, and enforces it on all lines changed.